src/AIS.cpp

```diff
@@ -84,7 +88,7 @@ ais_base::Position AIS::getPosition(ais::message_01 const& message) {
     );
     position.high_accuracy_position = message.get_position_accuracy();
     position.yaw = optionalAngleToRock(message.get_hdg()) * -1;
-    position.speed_over_ground = optionalFloatToRock(message.get_sog());
+    position.speed_over_ground = knots_to_ms(optionalFloatToRock(message.get_sog()));
```

Comment:
there is really no necessity to use a method

```suggestion
    position.speed_over_ground = optionalFloatToRock(message.get_sog()) * AIS::KNOTS_TO_MS;
```